make/modules/jdk.jdwp.agent/Lib.gmk

```diff
@@ -68,14 +68,13 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBJDWP, \
     DISABLED_WARNINGS_clang_threadControl.c := unused-but-set-variable \
         unused-variable, \
     DISABLED_WARNINGS_clang_utf_util.c := unused-but-set-variable, \
+    LDFLAGS := $(ICONV_LDFLAGS), \
     EXTRA_HEADER_DIRS := \
         include \
         libjdwp/export \
         java.base:libjava, \
     JDK_LIBS := java.base:libjvm, \
-    LIBS_linux := $(LIBDL), \
```

Comment:
Something got wrong here. You should have kept the LIBS_linux line, and the new LIBS line should be `LIBS := $(ICONV_LIBS)`.